src/components/main-form.js

```diff
@@ -80,22 +81,44 @@ const AmortizationTable = styled(Table)`
     border-left-color: ${GRAY}};
   }
 `
+const emphasize = cx('card', css(`border-color: ${LIGHT_GREEN};`))
+
+const box = bestOption => css`
+  background-color: ${bestOption ? LIGHT_GREEN : GRAY};
+  font-size: 3rem;
+  color: white;
+`
 
 /**
  * @param {*} props
- *   @property {boolean} highlight
+ *   @property {boolean} bestOption
  *   @property {Object} option
  */
 function ScenarioCol (props) {
-  const highlightClass = cx('card', css(`border-color: ${LIGHT_GREEN};`))
   return (
-    <Col className={`p-4 ${props.highlight && highlightClass}`}>
+    <Col className={`p-4 ${props.bestOption && emphasize}`}>
       <h5>{props.option.term} year scenario:</h5>
       <ol className={css('padding-inline-start: 1rem;')}>{props.children}</ol>
     </Col>
   )
 }
 
+/**
+ * @param {*} props
+ *   @property {boolean} bestOption
+ *   @property {Object} option
+ */
+function BoxCol (props) {
+  return (
+    <Col className={`p-4 text-center ${props.bestOption && emphasize}`}>
+      <div className={classNames(box(props.bestOption), 'p-3')}>
+        <b>{props.option.term}yr</b>
+      </div>
+      {props.bestOption && <h3>Better by {formatMoney(props.optCost)}</h3>}
+    </Col>
+  )
+}
```

Comment:
non-blocking: Something I'm now wondering w/ the design now that the summary also has highlights, should the two be one? 

If so, where should we place the explanation?

Reply:
I personally like them separate, we could run new designs and get more feedback later on